example_school_library_decorator/book.rb

```diff
@@ -1,7 +1,24 @@
+# rubocop:disable all
+
 class Book
+  @@books = []
   attr_accessor :title, :author, :rentals
+  def self.add_books(book)
+    @@books << book
+  end
+
+  def self.list_all_books
+    if @@books.empty?
+      puts 'No books found'
+      return
+    end
+
+    @@books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
 
-  def initialize(title)
+  def initialize(title, author)
     @title = title
     @author = author
     @rentals = []
```

Comment:
- [ ] Kindly intialize your constructor right after defining your `attr_accessors`, `attr_writers` or `attr_readers`, this is considered good practice as it will make your code be more readable.